pages/common/git-commit-graph.md

```diff
@@ -0,0 +1,16 @@
+# git commit-graph
+
+> Write and verify Git commit-graph files.
+> More information: <https://git-scm.com/docs/git-commit-graph>.
+
+- Write a commit-graph file for the packed commits in the repository's local `.git` directory:
+
+`git commit-graph write`
+
+- Write a commit-graph file containing all reachable commits:
+
+`git show-ref --hash | git commit-graph write --stdin-commits`
+
+- Write a commit-graph file containing all commits in the current commit-graph file along with those reachable from `HEAD`:
```

Comment:
This sounds a bit confusing to me. Can you explain what this does please?

Reply:
Sure!  That came straight from the [git docs](https://git-scm.com/docs/git-commit-graph).

Reply:
Right, but which part? Reading all of the docs for this might take a long time...?

Reply:
>Right, but which part? 

https://git-scm.com/docs/git-commit-graph

>Besides, if this came fromt he docs, it doesn't sound like the docs are particularly clear on the subject.

You're definitely right about that.  I think they focus more on Git itself :)

Reply:
Now, actually, I don't really even know what they intended there, so I don't know if I can improve it! ;P

I'll try though.

Reply:
Oh, I see. `giit show-ref -s` seems to list a number of different commit hashes, which are then processed by the `commit-graph` subcommand.